### ISSUE TITLE
Make error: ./src/xml2ps.c:2270:5: error: call to undeclared function 'get_BDIR'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]     get_BDIR(BDIR, ps_image_par);

### ISSUE THREAD

**Issue by TakumiOtagaki** [rcsb/RNAView#3]
I did the following commands, 
```
git clone <url>
cd RNAview
make
```

then, error occured:

```
./src/xml2ps.c:2270:5: error: call to undeclared function 'get_BDIR'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
    get_BDIR(BDIR, ps_image_par);
```

I checked `RNAview/include/xml2ps` and find that get_BDIR function isn't declared.
So after adding 
```
void getBDIR(char *BDIR, char *parfile);
```
in line 18 in RNAview/include/xml2ps.h, 
it works.

**Comment by mmagnus** [comment]
```
./src/xml2ps.c:1695:39: warning: format specifies type 'int' but the argument has type 'long' [-Wformat]
                    xy2[1], xy2[2]-4, author_seq[k1]);
                                      ^~~~~~~~~~~~~~
./src/xml2ps.c:2279:5: error: call to undeclared function 'get_BDIR'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
    get_BDIR(BDIR, ps_image_par);
    ^
30 warnings and 1 error generated.
make: *** [obj/xml2ps.o] Error 1
``` 

the same here

**Comment by trprcll** [comment]
Fixes that address compilation errors have been merged into master, and included in [release v2.0.0](https://github.com/rcsb/RNAView/releases/tag/RNAView-v2.0.0) that was created today.

**Comment by mmagnus** [comment]
works! thanks!
